### ISSUE TITLE
error while processing mouse data

### ISSUE THREAD

**Issue by kvshams** [neurogenomics/MAGMA_Celltyping#42]
getting zero gene ortholog 

```
ctd2 = prepare.quantile.groups(ctd,specificity_species = "mouse",
                               gwas_species = "human", numberOfBins=40)
```
```
[1] "Dropping all genes that do not have 1:1 homologs between the two species"
[1] "Species for which homology classes are available:"
character(0)
[1] "Available species options:"
[1] all    simple
<0 rows> (or 0-length row.names)
Error in check.species(requestedSpecies = species1, allSpecies = allHomologs$`Common Organism Name`): ERROR: species does not match available options
Traceback:

1. prepare.quantile.groups(ctd, specificity_species = "mouse", gwas_species = "human", 
 .     numberOfBins = 40)
2. One2One::analyse.orthology(specificity_species, gwas_species, 
 .     allHomologs)   # at line 7 of file <text>
3. check.species(requestedSpecies = species1, allSpecies = allHomologs$`Common Organism Name`)
4. stop("ERROR: species does not match available options")
5. 
```
While checking the code,  it may be associated with `One2One` packages `load.homologs.r`

```
    hom_vert = read.table("http://www.informatics.jax.org/downloads/reports/HOM_AllOrganism.rpt",sep="\t",stringsAsFactors = FALSE,quote="") ```

 as the column header has been changed. Not sure what was the previous and how that column handled for the downstream for debugging.

**Comment by Al-Murphy** [comment]
Hi @kvshams,

Thanks for letting us know, I am able to replicate the error and am looking into a fix now. I'll update when it is resolved.

Kind regards,
Alan.

**Comment by kvshams** [comment]
@Al-Murphy Thanks.... it worked out well with just fixing the column name and it worked well. I included those changes in the pull at `One2One`.

**Comment by Al-Murphy** [comment]
Thanks @kvshams, we are investigating to see if the data has changed along with the column header, it appears there is a significant difference in the new dataset size. If you're interested, see the issue here:

https://github.com/NathanSkene/One2One/issues/3

**Comment by Al-Murphy** [comment]
This issue has been resolved thanks to a fix to the OnetoOne package. @bschilder can you confirm if you plan to replace the use of OnetoOne down the line in your MAGMA_celltyping branch? @NathanSkene mentioned you might?

**Comment by bschilder** [comment]
@Al-Murphy sorry for the late reply, these were all getting sent to my spam folder. 

Yes, I am planning on updating `prepare.quantile.groups()` to take advantage of the upgrades in interspecies gene conversion in the DelayedArray branch of EWCE
